content/post/2020-12-31-modern-blogdown/index.Rmarkdown

```diff
@@ -0,0 +1,733 @@
+
+{{% alert note %}}
+You can also delete a widget file. You can always recover `*.md` widget files by going into your `themes/hugo-academic/exampleSite/content/home/` folder.
+{{% /alert %}}
+
+For my own site, I use 4 main home page widgets:
+
+1. [*about*](/#about) (photo / icons / bio / interests / education)
+1. [*slider*](/#slider) (used to showcase some feedback from my workshops)
+1. [*posts*](/#posts) (set to show only the most recent)
+1. [*talks*](/#talks) (set to show only the most recent)
+
+
+### Transplant widgets
+
+If you opted for a more streamlined home page with fewer widgets, you may experience *widget pruning regret*. Many are very useful, and you may wish to use widgets on *other* pages that are not the home page. In this theme, this is possible! Even if you turn off a widget on the home page, you can create what is called a widget page and add or even combine widgets there. I make heavy use of widget pages in my own site. Here are the steps (following the [docs](https://sourcethemes.com/academic/docs/managing-content/#create-a-widget-page)):
+
+1. Create a new folder in `content/`; let's call it `resume`
+
+2. Inside `content/resume/`, add a file named `index.md`
+
+3. Populate `content/resume/index.md` with a YAML, like this:
+
+    ```yaml
+    ---
+    summary: More about my work experience
+    title: "Resume"
+    type: widget_page
+    ---
+    ```
+    
+{{% alert note %}}
+The very critical line here is the `type: widget_page`---this sets you up to now copy over widgets from `content/home/` in this new folder. 
+{{% /alert %}}
+
+4. Copy the widget `*.md` files you'd like to use into this `content/resume/` folder. Edit their metadata (weights, other info), and be sure to set `active = true`. For my own resume, I copied the *experience* and *accomplishments* widgets over.
+
+5. If you want to access this new widget page from your top navbar, open up your `config/_default/menus.toml` and add it there, like:
+
+    ```toml
+    [[main]]
+      name = "Resume"
+      url = "/resume"
+      weight = 50
+    ```
+    
+{{% alert note %}}
+You can link to any specific widget by taking your baseurl and appending `/#{name-of-widget}`, so `/#slider` links to [*my homepage slider*](/#slider). And `/resume/#accomplishments` links to [*my honors & awards*](/resume/#accomplishments)
+{{% /alert %}}
+
+Let's run one final check, which wraps all 5 checking functions we've used so far into a single final checklist:
+
+```
+> blogdown::check_site()
+```
+
+This is immensely comforting to me; read more about these new checking functions [here](/post/2020-12-27-blogdown-checks/).
+
+![](https://media.giphy.com/media/j1zW1Ng86MV78tRLOo/giphy.gif)
+
+## We made it!
+
+At this point, you should be up and running with `hugodown`, GitHub, and Netlify. You now have the scaffold up and ready for your ideas, your style, and your voice. If you made it this far, please share your site- I'd love to see it!
```

Comment:
I guess here is a typo:

```suggestion
At this point, you should be up and running with `blogdown`, GitHub, and Netlify. You now have the scaffold up and ready for your ideas, your style, and your voice. If you made it this far, please share your site- I'd love to see it!
```

This is really an excellent tutorial! You made it again! Many thanks!

Reply:
ack thank you! The number of times I typed Hugo in that post- thank you so much for the catch 😹